lucene/core/src/java/org/apache/lucene/search/KnnByteVectorQuery.java

```diff
@@ -72,14 +72,30 @@ public KnnByteVectorQuery(String field, byte[] target, int k) {
    * @throws IllegalArgumentException if <code>k</code> is less than 1
    */
   public KnnByteVectorQuery(String field, byte[] target, int k, Query filter) {
-    super(field, k, filter);
+    this(field, target, k, filter, null);
+  }
+
+  /**
+   * Find the <code>k</code> nearest documents to the target vector according to the vectors in the
+   * given field. <code>target</code> vector.
+   *
+   * @param field a field that has been indexed as a {@link KnnFloatVectorField}.
```

Comment:
```suggestion
   * @param field a field that has been indexed as a {@link KnnByteVectorField}.
```
copy/paste from existing line 68 above -- #13637 for that.